tests/unauthorized/sharingTest.js

```diff
@@ -49,7 +49,7 @@ describe('Unauthorized: Currently testing file/folder sharing,', function () {
     })
   })
 
-  it('checking method : shareFileWithGroup', function (done) {
+  fit('checking method : shareFileWithGroup', function (done) {
```

Comment:
these can be skipped if we enable all the tests in the file. fdescribe works for all in this kind of cases